pyproject.toml

```diff
@@ -123,3 +123,16 @@ target-version = ['py38']
 [tool.check-wheel-contents]
 # W009: Wheel contains multiple toplevel library entries
 ignore = "W009"
+
+[tool.ruff]
+src = ["pytest"]
+select = [
+    "I",  # isort
+]
+
+[tool.ruff.lint.isort]
+force-single-line = true
+known-local-folder = ["pytest", "_pytest"]
+known-third-party = ["test_source", "test_excinfo"]
+force-sort-within-sections = true
```

Comment:
reorder-imports separates `import`s and `from`s, so if want the same style we should keep it false (the default) no?

Reply:
Basically made some blind try to go around https://github.com/astral-sh/ruff/issues/4670 but the comment were not specific enough to be actionable, let's remove.